docs/HowTo/Logging and Errors.md

```diff
@@ -0,0 +1,168 @@
+# Logs & Errors Consolidation
+Although there is sufficient logging in Quorum (geth), the error messages are not always straightforward to decipher.
+This is our take on cataloguing of error messages along with possible cause and remediation actions, to serve as a first point of reference before reaching out to the support team.
+
+!!! note
+    The log level “ERROR” is written out in uppercase as part of the log message and can be used for alert monitoring.
+
+| Message & Parameters | Cause | Action |
+| -------------------- | ----- | ------ |
+| "Could not search for pattern", "pattern", pattern, "contract", contracts[types[i]], "err", err | Generated by `abigen` due to an internal error when generating ABI for a contract |  *Raise an issue ticket* |
+| "account listing failed", "error", err | Call to Clef API method `account_list` failed |  *Depends on the error detail in the log message* |
+| "operation SelfDerive not supported on external signers" | An unsupported operation was performed by an external wallet.  | |
```

Comment:
In case of empty action cell, what is the user expected to do? If no action is possible, is it worth having this in the doc?

Reply:
I don't want to clutter it with loads of 'no action needed/possible' statements, so I've just changed the intro slightly to make it a bit clearer that not every entry has a suggested action.

Reply:
Yes but then if I reach this error message, what is the doc telling me to do? It's going to lead to frustration. Can we say something like "contact the 3rd party software vendor" or something just to make sure users don't get stuck or think that it's an issue on our side and we should do something?